src/ice/agent.rs

```diff
@@ -365,6 +367,9 @@ impl IceAgent {
             }
         }
 
+        // "Adopt" any incoming candidate by setting our current ufrag.
+        c.set_ufrag(&self.local_credentials.ufrag);
```

Comment:
How was this getting set before?

Reply:
Not sure it was other than parsed from the SDP.

Reply:
From the remote? But it returns its won clients, not ours